project/Http4sPlugin.scala

```diff
@@ -153,12 +153,12 @@ object Http4sPlugin extends AutoPlugin {
     val ip4s = "2.0.4"
     val javaWebSocket = "1.5.3"
     val jawn = "1.3.2"
-    val jawnFs2 = "1.2.0"
+    val jawnFs2 = "1.2.1"
     val jetty = "9.4.46.v20220331"
-    val keypool = "0.3.5"
+    val keypool = "0.3.6"
```

Comment:
I'm not sure keypool made it.
https://repo1.maven.org/maven2/org/typelevel/keypool_3/